contrib/systemd/mako.service.in

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=A lightweight Wayland notification daemon
+Documentation=man:mako(1)
+PartOf=wayland-session.target
+
+[Service]
+Type=dbus
+BusName=org.freedesktop.Notifications
+ExecStart=@bindir@/mako
+
+[Install]
+WantedBy=wayland-session.target
```

Comment:
Is this a known supported target? My Arch Linux installation does not ship this target by default.

Reply:
I'm open to suggestions

it's shipped in this aur package https://aur.archlinux.org/packages/sway-services-git

But there are not many choices currently. Some of my thoughts were `sway-session.target` or `graphical-session.target`.

Reply:
So `sway-services` would be a dependency of this package? TBH, this seems like a forced dependency, and added complexity for many setups that don't really need this.